girder/cumulus/server/models/aws.py

```diff
@@ -132,12 +133,14 @@ def validate(self, doc):
 
         return doc
 
-    def create_profile(self, userId, name, access_key_id, secret_access_key,
-                       region_name, availability_zone, public_ips):
+    def create_profile(self, userId, name, profile_type, access_key_id,
+                       secret_access_key, region_name, availability_zone,
+                       public_ips):
 
         user = getCurrentUser()
         profile = {
             'name': name,
+            'type': profile_type,
             'accessKeyId': access_key_id,
```

Comment:
What is the type field for? Aren't these always 'ec2' profiles? If this is part of a move to make the profiles more generic ( with a view to other providers in the future ) I think we should look at some renaming ...